BankSDK/GiniBankSDKExample/GiniBankSDKExample/SettingsViewController.swift

```diff
@@ -1,115 +1,223 @@
 //
 //  SettingsViewController.swift
-//  Example Swift
+//  GiniBankSDKExample
 //
-//  Created by Nadya Karaban on 19.02.21.
+//  Created by Valentina Iancu on 07.06.23.
 //
 
-import Foundation
 import UIKit
 import GiniCaptureSDK
 import AVFoundation
 
 protocol SettingsViewControllerDelegate: AnyObject {
-    func settings(settingViewController: SettingsViewController,
-                  didChangeConfiguration captureConfiguration: GiniConfiguration)
+	func settings(settingViewController: SettingsViewController,
+				  didChangeConfiguration captureConfiguration: GiniConfiguration)
 }
 
-final class SettingsViewController: UIViewController {
-    
-    weak var delegate: SettingsViewControllerDelegate?
-    var giniConfiguration: GiniConfiguration!
+class SettingsViewController: UIViewController {
 
-    @IBOutlet weak var fileImportControl: UISegmentedControl!
-    @IBOutlet weak var openWithSwitch: UISwitch!
-    @IBOutlet weak var qrCodeScanningSwitch: UISwitch!
-    @IBOutlet weak var multipageSwitch: UISwitch!
-    @IBOutlet weak var flashToggleSwitch: UISwitch!
-    @IBOutlet weak var bottomBarSwitch: UISwitch!
-    @IBOutlet weak var onlyQRCodeScanningSwitch: UISwitch!
-
-    @IBAction func fileImportOptions(_ sender: UISegmentedControl) {
-        switch sender.selectedSegmentIndex {
-        case 0:
-            giniConfiguration.fileImportSupportedTypes = .none
-        case 1:
-            giniConfiguration.fileImportSupportedTypes = .pdf
-        case 2:
-            giniConfiguration.fileImportSupportedTypes = .pdf_and_images
-        default: return
-        }
-        
-        delegate?.settings(settingViewController: self, didChangeConfiguration: giniConfiguration)
+	@IBOutlet private weak var navigationBarItem: UINavigationItem!
+	@IBOutlet private weak var navigationBar: UINavigationBar!
+	@IBOutlet private weak var tableView: UITableView!
+	
+	var sectionData = [SectionType]()
+	let giniConfiguration: GiniConfiguration
+	
+	weak var delegate: SettingsViewControllerDelegate?
+	
+	// MARK: - Initializers
+	
+	init(giniConfiguration: GiniConfiguration) {
+		self.giniConfiguration = giniConfiguration
+		super.init(nibName: nil, bundle: nil)
+	}
+	
+	required init?(coder aDecoder: NSCoder) {
+		fatalError("init(coder:) has not been implemented")
+	}
+	
+	override func viewDidLoad() {
+        super.viewDidLoad()
+		setupTableView()
```

Comment:
[Suggestion]: I would unite 2 method setupTableView() and setContent() into configureTableView()
styleNavigationBar() and addNavigationBarItems() into configureNavigationBar()